### ISSUE TITLE
[enterprise-4.9] Issue in file authentication/ldap-syncing.adoc

### ISSUE THREAD

**Issue by TiloGit** [openshift/openshift-docs#39286]
### Which section(s) is the issue in?
https://docs.openshift.com/container-platform/4.9/authentication/ldap-syncing.html#ldap-auto-syncing_ldap-syncing-groups

cronjob YAML: **Example ldap-sync-cron-job.yaml**
 `             image: "openshift/origin-cli:latest"`

### What needs fixing?
the linked oc client image on docker is 3years old (BuildDate:"2019-04-10T12:24:28Z")
https://hub.docker.com/r/openshift/origin-cli/tags

```console
joe@server1~ $ docker run --rm -it openshift/origin-cli:latest  bash -c "oc version"
Client Version: version.Info{Major:"4", Minor:"0+", GitVersion:"v4.0.0-alpha.0+af45cda-1969-dirty", GitCommit:"af45cda", GitTreeState:"", BuildDate:"2019-04-10T12:24:28Z", GoVersion:"", Compiler:"", Platform:""}
```

should one from 
`image: registry.redhat.io/openshift4/ose-cli`
https://catalog.redhat.com/software/containers/openshift4/ose-cli/5cd9ba3f5a13467289f4d51d

or quay.io be used? 
https://quay.io/repository/openshift/origin-cli

**Comment by bergerhoffer** [comment]
@soltysh Do you know the right image that we should be using here?

**Comment by soltysh** [comment]
@bergerhoffer I'd go with `registry.redhat.io/openshift4/ose-cli` since I've noticed we use that registry in other places as well.

**Comment by bergerhoffer** [comment]
Thanks @soltysh, I'll open a PR for that update. Should we still use the `latest` tag? Like:

```
image: "registry.redhat.io/openshift4/ose-cli:latest"
```

**Comment by bergerhoffer** [comment]
PR is up: #39608

**Comment by TiloGit** [comment]
> Thanks @soltysh, I'll open a PR for that update. Should we still use the `latest` tag? Like:
> 
> ```
> image: "registry.redhat.io/openshift4/ose-cli:latest"
> ```

I think should be same version (min +/-1) as OCP cluster.

**Comment by bergerhoffer** [comment]
If we did the same version then looks like the tag would be `v4.9` for 4.9 (of course, and updated per other versions as appropriate).

@soltysh Let me know if I should update the tag from `latest` to `v4.10` / `v4.9`, etc. The only one I can't see from https://catalog.redhat.com/software/containers/openshift4/ose-cli/5cd9ba3f5a13467289f4d51d is `v4.7`, so just let me know if that would be okay to use for 4.7.

**Comment by soltysh** [comment]
@bergerhoffer I'd just go with `:latest` this will allow everyone interested to get started, and later on they are hopefully knowledgeable to use appropriate tag matching their OCP version. Using specific version would require us to update it frequently, which is not desirable, imo.

**Comment by bergerhoffer** [comment]
Fixed via #39608, updates will be live shortly.
